examples/example.cpp

```diff
@@ -171,7 +171,7 @@ int main()
     });
 
     // enables all log
-    app.loglevel(crow::LogLevel::DEBUG);
+    app.loglevel(crow::LogLevel::Debug);
```

Comment:
Are the uppercase names part of the compatibility issues?

Reply:
Yes, because the uppercase names won't exist on an MSVC build as they are disabled by this:

```cpp
namespace crow
{
    enum class LogLevel
    {
#ifndef ERROR
        DEBUG = 0,
        INFO,
        WARNING,
        ERROR,
        CRITICAL,
#endif

        Debug = 0,
        Info,
        Warning,
        Error,
        Critical,
    };
```

Reply:
True but these examples are for Linux, in fact, they don't even get built on Windows. The one example for windows is example_vs (Visual Studio)

Reply:
Well silly me, in my ignorance I made all the linux examples work on windows too.

Though in fairness, it is not uncommon for projects to define macros such as DEBUG or ERROR for their own logging purposes.  Though I too enjoy using upper-case enums, there is a tradition that uppercase be used for #define macros.

Also in fairness, Microsoft do horrific things like `#define min(X, Y)` and put it in their default libraries.

Reply:
It might be best in this case to just remove the uppercase definitions and change the documentation to properly reflect that, but that can be done in another PR.